GitUI/GitExtensionsForm.cs

```diff
@@ -285,16 +285,16 @@ private void RestorePosition(String name)
         private static Rectangle? FindWindowScreen(Point location)
         {
             SortedDictionary<float, Rectangle> distance = new SortedDictionary<float, Rectangle>();
-            foreach (var rect in (from screen in Screen.AllScreens
-                                  select screen.WorkingArea))
+            foreach (var rect in from screen in Screen.AllScreens
+                                  select screen.WorkingArea)
```

Comment:
:bulb: This could be filed as a StyleCop Analyzers bug. It should have corrected the indentation of `select`.